test/HostActivationTests/project.json

```diff
@@ -3,7 +3,7 @@
   "dependencies": {
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0"
+      "version": "1.0.2"
```

Comment:
@gkhanna79 is "1.0.0-preview2-003133" exactly the string to pass for the '--channel' arg ('--channel 1.0.0-preview2-003133')? Previously we were using 'rel-1.0.0' which seems to suggest a different string format